kolibri/plugins/learn/assets/src/my_downloads/views/DownloadsList/index.vue

```diff
@@ -239,18 +234,35 @@
         },
       },
       paginatedDownloads() {
-        const startIndex = (this.currentPage - 1) * this.itemsPerPage;
-        const endIndex = startIndex + this.itemsPerPage;
-        return this.downloads.slice(startIndex, endIndex);
+        if (this.downloads.length > 0) {
+          const startIndex = (this.currentPage - 1) * this.itemsPerPage;
+          const endIndex = startIndex + this.itemsPerPage;
+          return this.downloads.slice(startIndex, endIndex);
+        } else {
+          return [];
+        }
+      },
+      downloadItemsListLength() {
+        return Object.keys(this.downloadRequestMap).length;
+      },
+      totalPageNumber() {
+        if (
+          this.downloadsToDisplay &&
```

Comment:
This is no longer defined - I think this is just `this.downloads` now.

Also, this prop confuses me, because I am not sure what it is calculating.